### PR TITLE
address uneccessary dynamic shapes in argmax decomposition

### PR DESCRIPTION
**Pull request by zjgarvey**
Addresses <https://github.com/iree-org/iree/issues/19262#issue>

### CODE REVIEW

File: lib/Dialect/Torch/Transforms/DecomposeComplexOps.cpp [hunk 1]
```diff
@@ -2593,16 +2593,22 @@ class DecomposeAtenArgMinMaxOp : public OpRewritePattern<OpTy> {
...
     // happens on the 0th dimension.
     if (isa<Torch::NoneType>(dim.getType())) {
-      BaseTensorType flattenType =
-          cast<BaseTensorType>(inputType.getWithSizesAndDtype(
-              {kUnknownSize}, inputType.getOptionalDtype()));
-      Value zero =
-          rewriter.create<ConstantIntOp>(loc, rewriter.getI64IntegerAttr(0));
-      Value end = rewriter.create<ConstantIntOp>(
-          loc, rewriter.getI64IntegerAttr(inputRank - 1));
+      Value zero = rewriter.create<ConstantIntOp>(loc, 0);
```
Comment (jinchen62) [inline lib/Dialect/Torch/Transforms/DecomposeComplexOps.cpp:2596]: Should the 0 be `rewriter.getI64IntegerAttr(0)`?

Comment (zjgarvey) [reply lib/Dialect/Torch/Transforms/DecomposeComplexOps.cpp:2596]: The builder allows passing an integer instead of an integer attr.

**Review by jinchen62 — APPROVED** [review]
LGTM

**Review by AmosLewis — APPROVED** [review]
lgtm

**Pull request merged**